src/data/spotlight/paul-maddern.md

```diff
@@ -0,0 +1,136 @@
+---
+name: "Paul Maddern"
+role: "Web Platform Consultant"
+date: "2025-04-15"
```

Comment:
Would you mind if we schedule an interview with you for 2025-05-02, Friday please?